### ISSUE TITLE
[Bug] x-casaos.tagline displayed twice

### ISSUE THREAD

**Issue by mr-manuel** [IceWhaleTech/CasaOS#1342]
**Describe the bug**

> A clear and concise description of what the bug is.

In the AppStore the `x-casaos.tagline` is displayed twice.

**To Reproduce**

> Steps to reproduce the behavior:

1. Open the AppStore
2. Click on any app

**Expected behavior**

> A clear and concise description of what you expected to happen.

The tagline should only displayed once.

**Screenshots**

> If applicable, add screenshots to help explain your problem.

![grafik](https://github.com/IceWhaleTech/CasaOS/assets/12592765/36bb30fa-a120-4312-aaff-8947d4c481a1)

**Comment by CorrectRoadH** [comment]
@ETWang1991 Is this a bug or a feature?👀

**Comment by ETWang1991** [comment]
@mr-manuel There are indeed some issues here, but I personally think that the screenshot and text below need better overall typography

**Comment by mr-manuel** [comment]
Yes, I opened another issue for the typography: https://github.com/IceWhaleTech/CasaOS-AppStore/issues/255

**Comment by jerrykuku** [comment]
We will add this feature in the next release.
